modules/swagger-codegen/src/main/resources/swift3/AlamofireImplementations.mustache

```diff
@@ -76,8 +76,15 @@ open class AlamofireRequestBuilder<T>: RequestBuilder<T> {
      May be overridden by a subclass if you want to control the request
      configuration (e.g. to override the cache policy).
      */
-    open func makeRequest(manager: SessionManager, method: HTTPMethod, encoding: ParameterEncoding, headers: [String:String]) -> DataRequest {
-        return manager.request(URLString, method: method, parameters: parameters, encoding: encoding, headers: headers)
+    open func makeRequest(manager: SessionManager, method: HTTPMethod, encoding: ParameterEncoding, headers: [String:String]) throws -> DataRequest {
```

Comment:
Added `throws` here... which might break peoples implementation if they are extending `AlamofireRequestBuilder`. They would need to add some type of `try` in there code if they are calling `super.makeRequest`. I can back this out, but originally the manager.request was handling these errors for us.

Reply:
I think it would be cleaner to keep the original implementation w/o throws and just return an error if the try's below fail just like in manger.request:
            return request(originalRequest, failedWith: error)

Reply:
Thats what I wanted to do originally but that method is private. I could try figuring out whats going on in that function and try to mimic it?

Reply:
could this logic be pushed into SessionManager.swift? so AlamofireImplementations just calls a helper method in that class which then has access to that function.

Reply:
SessionManager is part of Alamo if I remember correctly. I don't think that will work.

Reply:
ah you're correct.  maybe just return some type of error instead (not necessary copying Alamo's logic exactly but something similar)

Reply:
So change the method signature? Right now it expects DataRequest to be returned. I was trying to avoid changing the method signature but if thats ok I can do that.

Reply:
@jgavris any thoughts as to how to avoid changing the method signature here so we all don't have to try/catch around these blocks?

Reply:
my only thought would be to make a new request in the catch block w/ some bogus url to force an error... but seems messy.

Reply:
I though about this for a while and I think I finally have a clean solution. I'll update the pull request in a little bit.